internal/pkg/daemon/seccompprofile/seccompprofile.go

```diff
@@ -122,8 +120,21 @@ func (r *Reconciler) Setup(
 
 // Healthz is the liveness probe endpoint of the controller.
 func (r *Reconciler) Healthz(*http.Request) error {
-	if !r.ready.Get() {
-		return errors.New("not ready")
+	return r.checkSeccomp()
+}
+
+// checkSeccomp verifies if the seccomp is supported by the node.
+func (r *Reconciler) checkSeccomp() error {
+	if !seccomp.IsSupported() {
+		err := errors.New("seccomp not supported")
+		err = fmt.Errorf("node %q: %w", os.Getenv(config.NodeNameEnvKey), err)
```

Comment:
nit: I'd merge both lines into something like 
```suggestion
		err := fmt.Errorf("seccomp not supported on node %q", os.Getenv(config.NodeNameEnvKey))
```

Reply:
They were merged together in the previous commit but it was a lint warning. I split them in order to fix that warning. See https://github.com/kubernetes-sigs/security-profiles-operator/runs/5498882895?check_suite_focus=true#step:5:114